src/test/java/org/minions/devfund/lourdes/MovieDatabaseTest.java

```diff
@@ -0,0 +1,161 @@
+package org.minions.devfund.lourdes;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Implemented tests for {@link MovieDatabase}.
+ */
+public class MovieDatabaseTest {
+
+    private MovieDatabase movieDatabase;
+    private String movieName1;
+    private String[] actors1;
+
+    /**
+     * Method that setup all initial values.
+     */
+    @Before
+    public void setup() {
+        movieDatabase = new MovieDatabase();
+        movieName1 = "Sexto sentido";
+        actors1 = new String[]{"Bruce Willis", "Haley Joel"};
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Bruce Willis" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)